lib/core/scenic_events_definitions.ex

```diff
@@ -192,7 +192,7 @@ defmodule ScenicWidgets.ScenicEventsDefinitions do
       @period {:key, {:key_dot, @key_pressed, []}}
       @bang {:key, {:key_1, @key_pressed, [:shift]}}
       @question_mark {:key, {:key_slash, @key_pressed, [:shift]}}
-      @asperand {:key, {:key_2, @key_pressed, [:shift]}}
```

Comment:
I love all your other changes, but this was was actually intentionally `asperand` http://editingmonks.blogspot.com/2010/02/ampersand-vs-asperand.html

I used to have just `at_sign` and can change it back to that if asperand is confusing

Reply:
Ah, TIL. Changed back in: 974e0573eb0e4e6a08b74e64884ef9f8b63d1201